flight/Modules/UAVOMavlinkBridge/UAVOMavlinkBridge.c

```diff
@@ -125,7 +123,8 @@ static int32_t uavoMavlinkBridgeInitialize(void) {
 		module_enabled = true;
 		updateSettings();
 
-		serial_buf = PIOS_malloc(MAVLINK_MAX_PACKET_LEN);
+		mav_msg = PIOS_malloc(sizeof(*mav_msg));
```

Comment:
@tracernz good catch. I think in general we throw a PIOS_Assert. It's a really big deal if the malloc fails.